### PR TITLE
Add clubbed tail and tendril to base weapon types localization

### PR DESCRIPTION
**Pull request by ammalagonc**
For Animal Companion - Sundaflora
Edit - also added tendril

### CODE REVIEW

File: static/lang/en.json [hunk 1]
```diff
@@ -5243,6 +5243,7 @@
                 "claw": "Claw",
                 "claw-blade": "Claw Blade",
                 "club": "Club",
+                "clubbed-tail": "Clubbed tail",
```
Comment (simonward) [inline static/lang/en.json:5246]:
```suggestion
                "clubbed-tail": "Clubbed Tail",
```

File: static/lang/en.json
Comment (pedrogrullada) [inline static/lang/en.json:1]: Should these really be here? You'd be adding it as a base weapon type, which surely this isn't

Comment (ammalagonc) [reply static/lang/en.json:1]: Any suggestions as to where they should go?

Comment (pedrogrullada) [reply static/lang/en.json:1]: Depends on what they're for. If it's for the companion compendia module, I'd leave it unlocalised for now and wait for that module to have its own localization file

**Pull request closed without merging**